server/types.go

```diff
@@ -52,3 +53,46 @@ type esSearchResponseHit struct {
 	Highlights json.RawMessage `json:"highlight"`
 	Sort       []interface{}   `json:"sort"`
 }
+
+type esMultiGetRequest struct {
```

Comment:
In my next pr, i'll be getting rid of these in favor of the esutils package. Anything here that is not there, I will port over.
You don't need to worry about it, I'll take care of it